src/jpegParser.js

```diff
@@ -74,15 +69,22 @@
       previousByte = currentByte;
       offset += 1;
     }
-    return offset + 1;
+    return offset - 1;
```

Comment:
I need to refresh the details of the exif spec. Why did you replace - 1 with +1? I just want to make sure that it's not a mistake.